cfgov/unprocessed/js/routes/external-site.js

```diff
@@ -53,7 +53,7 @@ function ExternalSite(element) {
   function _updateContent() {
     const plurality = _duration === 1 ? '' : 's';
     const content =
-      "<span class='external-site_reload-duration'>" +
+      "<span style='line-height:0' class='h2'>" +
```

Comment:
What would be the consequence of removing the `line-height` entirely?

Reply:
It causes the line to jump/shift when the first number is added. This could likely be worked around by putting some default content in this span in the actual template, but that would be more text than just adding this line-height.